troubleshooting.md

```diff
@@ -0,0 +1,550 @@
+# Troubleshooting Guide for BM IPI Deployments
```

Comment:
I would set all other titles to level '2' via ## so that it follows regular article structure

Reply:
Leaving # for main title and ## for everything else

Reply:
Adjusted the headers as suggested, Thanks.